openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -31,6 +38,116 @@ def get_identity(self, data):
         except AttributeError:
             return None
 
+class AccountFullUserProfileReadOnlySerializer(serializers.Serializer):
```

Comment:
You need an extra newline here. Pep8 is very particular:

https://www.python.org/dev/peps/pep-0008/#blank-lines